lib/types/operation.js

```diff
@@ -599,7 +600,7 @@ Operation.prototype.getBody = function (headers, args, opts) {
             bodyParam.append(key, value.value);
           } else {
             if (Array.isArray(value)) {
-              bodyParam.append(key, this.encodeQueryCollection(param.collectionFormat, key, value));
+              bodyParam.append(key, this.encodeQueryCollection(param.collectionFormat, key, value).split('=').slice(1).join('='));
```

Comment:
Well, the problem, it would duplicate the `key` in the first place.
See the serialization of the `bodyParam` object.
```js
expected: lang=deu,eng
actual:  lang=lang=deu&lang=eng 
```
and
```js
expected: country=DE&country=US
actual: country=country=DE&country=US 
```